pkg/apis/kubeone/v1beta3/conversion.go

```diff
@@ -30,3 +30,11 @@ func Convert_kubeone_KubeOneCluster_To_v1beta3_KubeOneCluster(in *kubeoneapi.Kub
 func Convert_v1beta3_ContainerRuntimeConfig_To_kubeone_ContainerRuntimeConfig(in *ContainerRuntimeConfig, out *kubeoneapi.ContainerRuntimeConfig, scope conversion.Scope) error {
 	return autoConvert_v1beta3_ContainerRuntimeConfig_To_kubeone_ContainerRuntimeConfig(in, out, scope)
 }
+
+// Convert_v1beta3_CiliumSpec_To_kubeone_CiliumSpec is an autogenerated conversion function.
```

Comment:
It's not autogenerated. :) And to be fair, I don't think we even need that comment.
```suggestion

```